packages/admin-ui/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "lib": [
+      "ES7",
+      "DOM"
+    ],
+    "target": "es5",
+    "allowJs": true,
+    "skipLibCheck": false,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true,
+    "jsx": "preserve"
+  },
+  "include": [
+    "src"
+  ],
+  "exclude": [
+    "src/tests"
```

Comment:
Why would we exclude tests?

Reply:
The nightwatch tests run on node, and does things like require-ing babel-register, which causes CRA to throw errors

Reply:
I'm wondering what this mean for unit tests, can they call their appropriate TS alternatives? Anyway, we can always iterate on that